airbyte-integrations/connector-templates/source-python-http-api/acceptance-test-config.yml.hbs

```diff
@@ -8,7 +8,7 @@ tests:
     - config_path: "secrets/config.json"
       status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
-      status: "exception"
+      status: "failed"
```

Comment:
This would make acceptance test to expect an exception in case of incorrect config, but in real case it it would generate FAIL message, not an exception.

Reply:
@avida Yes, but isn't it for another connector?